security.md

```diff
@@ -75,6 +76,14 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2018-14664"></a> Persisted XSS on all pages that use breadcrumbs
```

Comment:
The others include the CVE number in the title as well

Reply:
You are right, they do. Updated.